Tactical Paint (#6725)
FAForever/fa: changelog/snippets/features.6725.md

- (#6725) Tactical Paint feature:

    This feature adds ability to draw lines during game which teammates and observers can see.
    It is possible thanks to [this binary patch](https://github.com/FAForever/FA-Binary-Patches/pull/110).

    - Adds canvas mode where you can draw lines with color of player's army. This mode is enabled via keybind "Toggle Tactical Paint canvas" under "'Tactical Paint" section.
    - When in paint mode you can draw lines holding left mouse button and erase them holding right mouse button.
    - These drawings are shared between teammates.
    - You can mute specific player if one annoys you.

>     - Can be turned off completely **(it is by default, you have to turn this on to see!)**.

Reviewer: ```suggestion
    - Can be turned off completely (it is on by default).
```

>     - These drawings are left in replays, so, mind what you draw.

Reviewer: ```suggestion
    - These drawings are left in replays, so mind what you draw.
```

>     - You can adjust duration of **your** drawings with maximum of 1 minute.

Reviewer: Grammar fix suggestion, but really there should be details in this point or in an additional point on how long your teammates' or your own drawings stay on the map.
```suggestion
    - You can adjust the duration of **your** drawings up to a maximum of 1 minute.
```

>     - Observers see these between each over and can draw their own.

Reviewer: ```suggestion
    - Observers see all drawings and can draw their own.
```
